multicluster/config/integration/antrea-mc-member-access-sa.yml

```diff
@@ -0,0 +1,252 @@
+apiVersion: v1
```

Comment:
it would be better to generate this file via kustomize, otherwise, it will have no change when there is any antrea-multicluster-member role change, I will check this and probably to add a new PR with some of your codes. I will add you as co-author. thanks